src/Symfony/Installer/NewCommand.php

```diff
@@ -415,13 +415,19 @@ protected function generateComposerProjectName()
      */
     private function fixComposerPackageName($name)
     {
-        return strtolower(
-            preg_replace(
-                array('/([A-Z]+)([A-Z][a-z])/', '/([a-z\d])([A-Z])/'),
-                array('\\1-\\2', '\\1-\\2'),
-                strtr($name, '-', '.')
-            )
+        $name = preg_replace(
+            array('/([A-Z]+)([A-Z][a-z])/u', '/([a-z\d])([A-Z])/u'),
+            array('\\1-\\2', '\\1-\\2'),
+            strtr($name, '-', '.')
         );
+
+        if (function_exists('mb_strtolower')) {
+            $name = mb_strtolower($name);
+        } else {
+            $name = utf8_encode(strtolower(utf8_decode($name)));
```

Comment:
Actually, this is typically the kind of workaround I'm not sure about :confused: Do you have a better idea?